pkg/backend/openshiftcluster/create.go

```diff
@@ -322,11 +324,51 @@ func (m *Manager) Create(ctx context.Context) error {
 		return err
 	}
 
-	return i.Install(ctx, installConfig, platformCreds, image)
+	bootstrapLoggingConfig, err := m.getBootstrapLoggingConfig()
+	if err != nil {
+		return err
+	}
+
+	return i.Install(ctx, installConfig, platformCreds, image, bootstrapLoggingConfig)
 }
 
 var rxRHCOS = regexp.MustCompile(`rhcos-((\d+)\.\d+\.\d{8})\d{4}\-\d+-azure\.x86_64\.vhd`)
 
+func (m *Manager) getBootstrapLoggingConfig() (*bootstraplogging.Config, error) {
```

Comment:
I'm a bit antsy about the duplication with pkg/genevalogging.  I'm tempted to move this function into pkg/genevalogging so that the duplication is at least closer together.  At the same time there are a couple of easy refactors that could help (e.g. refactor adding a helper function for getting gcsKeyBytes, gcsCertBytes; make "AROClusterLogs" a const).

Reply:
Moved to a separate file in /pkg/genevalogging.go as a standalone function without a receiver. Moved the Namespace to a const.